.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
 
 script:
   - GO111MODULE=on go test ./...
-  - docker build --tag kube-bench .
+  - IMAGE_NAME=kube-bench make build-docker
```

Comment:
Is there any reason to keep IMAGE_NAME var outside of makefile?

Reply:
The use of the IMAGE_NAME is to make sure the line following that one continues to work.

Reply:
All right, I thought that the defaults hardcoded in makefile are enough, so when I run `make build-docker` it would work the same way as on Travis.